products/zoningtaxlots/seeds/_seeds.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+
+seeds:
+  - name: zonedist_priority
+    columns:
+      - name: zonedist
+        tests:
+          - not_null
+          - unique
+      - name: priority
+        tests:
+          - not_null
+          - unique
+      
```

Comment:
Just for tidyness, can you get rid of the whitespace on this line?

Other than that, looks great